gradle.properties

```diff
@@ -1,8 +1,7 @@
-org.gradle.vfs.watch=true
-org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.caching=true
 org.gradle.configuration-cache=true
+org.gradle.configureondemand=true
```

Comment:
can you tell us more about the changes to the gradle flags here and why there are being made?

Reply:
I see it was disabled by default, and suggest to be enabled in https://docs.gradle.org/8.2.1/userguide/build_environment.html#sec:gradle_configuration_properties

Reply:
I don't see a suggestion to enable this and this feature has no benefit in a single project build. Could you please undo this change.

Reply:
Done.

Reply:
Thanks!